internal/uvm/vsmb.go

```diff
@@ -13,25 +13,45 @@ import (
 
 const vsmbSharePrefix = `\\?\VMSMB\VSMB-{dcc079ae-60ba-4d07-847c-3493609c0870}\`
 
+// VsmbShare contains the host path for a Vsmb Mount
+type VsmbShare struct {
+	// UVM the resource belongs to
+	VM           *UtilityVM
```

Comment:
I dont believe the embedded vm is called anywhere from outside the uvm package so yes I think it can be unexported. And the same holds true for the other resources